buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -113,6 +119,7 @@
             logger.debug("-Dio.netty.allocator.normalCacheSize: {}", DEFAULT_NORMAL_CACHE_SIZE);
             logger.debug("-Dio.netty.allocator.maxCachedBufferCapacity: {}", DEFAULT_MAX_CACHED_BUFFER_CAPACITY);
             logger.debug("-Dio.netty.allocator.cacheTrimInterval: {}", DEFAULT_CACHE_TRIM_INTERVAL);
+            logger.debug("-Dio.netty.allocator.powerOfCacheLineSize: {}", DEFAULT_POWER_OF_CACHE_LINE_SIZE);
```

Comment:
@normanmaurer I think the terminology is `multiple of cache line size`. power would be `cachlinesize^n` if I am not mistaken.